lib/ZumoOta/src/WebServerCustom.cpp

```diff
@@ -69,9 +70,49 @@ WebServerCustom::~WebServerCustom()
 {
 }
 
+String WebServerCustom::listFiles(bool ishtml)

```

Comment:
indentation levels wrong, 2 instead of 4 spaces

Reply:
It will be taken care of by the next commit.